spec/factories/member.rb

```diff
@@ -4,7 +4,7 @@
     last_name "Pad"
     sex "female"
     marital_status "Widowed"
-    ssn "123 12 1234"
+    ssn "123121234"
```

Comment:
_not blocking_ - Since this data format has meaning, do we want to do something in the driver layer that removes the ` `s?

Reply:
Just realized that we include the `SocialSecurityNumber` concern  (which validates the format) in the relevant step classes but not the `Member` AR class. added a ticket for that here: https://trello.com/c/t4tYuREg/310-validate-format-of-ssn-on-member-class

Reply:
reverted factory change here since it was not relevant to this PR 25a6208